VA-Explorer/va_explorer
# Add field worker supervision interface

This PR implements changes needed to support a field worker supervision interface
- Adds "supervise_user" custom permission to Dashboard model
- Adds a new template to va_analytics for displaying a list of field workers and associate stats
  - Shows field worker stats such as their assigned locations, the number of submitted vas, the sum of those VAs errors/warnings, and estimated number of VAs per week
- Adds a novel method for estimating expected number of submitted VAs based on a weekly average over the past couple of months
- Adds ability to aggregate supervision stats by field worker or by facility
- Adds pivot functionality to new supervision table
- Adds tests to cover this new functionality

To test:
- login as an admin or data manager and click the supervision tab in the top ribbon
- confirm accurate information displayed
- also login as data viewer or field worker and confirm that this tab is not available

Closes #113
